django/chat/management/commands/eval_responses.py

```diff
@@ -269,7 +269,7 @@ def _test_qa_response(eval_instance, user):
                 )
             if "vector_store_table" in eval_instance:
                 library = Library.objects.get(
-                    vector_store_table=eval_instance["vector_store_table"]
+                    name=eval_instance["vector_store_table"]
```

Comment:
For clarity, the string "vector_store_table" should be replaced with "library_name" in all these files.

Reply:
Like, in the library model too? Just replace the string across the whole codebase?